apps/examples/src/e2e/TestingView.tsx

```diff
@@ -0,0 +1,154 @@
+import React, {
+  Component,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import {Platform, Text, View} from 'react-native';
+import * as RNSVG from 'react-native-svg';
+import ViewShot from 'react-native-view-shot';
+
+const address = ['ios', 'web'].includes(Platform.OS) ? 'localhost' : '10.0.2.2';
+const wsUri = `ws://${address}:7123`;
+
+const TestingView = () => {
+  const [renderedContent, setRenderedContent] = useState();
+  const [wsClient, setWsClient] = useState<WebSocket | null>(null);
+  const [readyToSnapshot, setReadyToSnapshot] = useState(false);
+  const wrapperRef = useRef<ViewShot>(null);
+  const [resolution, setResolution] = useState([600, 600]);
```

Comment:
If I understand correctly that `const [resolution, setResolution] = useState([600, 600]);` based on that [comment](https://github.com/software-mansion/react-native-svg/pull/2338/files#diff-c78b493fa10a00e2c85350027da97e6b7bbb15978657add1f56762148fd26c9fR11-R13), like `size * pixelratio`?

Reply:
Yes, those are just initial values, they are later replaced with values demanded by a client (jest server): 
https://github.com/software-mansion/react-native-svg/blob/b3ceb8fec9a5e8e92c1c787906e09aea255eb4ba/src/lib/__tests__/e2e/GeneralSvgRenderingTest.spec.tsx#L30-L35
Probably should use [0, 0] to avoid confusion or document it better.